cmd/state-installer/cmd.go

```diff
@@ -482,3 +476,41 @@ func determineLegacyUpdate(stateToolInstalled bool, packagedStateExe, payloadPat
 func noArgs() bool {
 	return len(os.Args[1:]) == 0
 }
+
+var branchRegex = regexp.MustCompile("Branch (\\S+)")
+var versionRegex = regexp.MustCompile("Version (\\S+)")
+
+func shouldUpdateInstalledStateTool() bool {
+	logging.Debug("Checking if installed state tool is an older version.")
+
+	stateExe := constants.StateCmd + exeutils.Extension
```

Comment:
Please use the path that was returned here: 

https://github.com/ActiveState/cli/pull/2053/files#diff-2dd975f5d9eec0de10d0d721cf03ea5c32713c0df58a631ce2f711d519575608R233

Reply:
That link doesn't take me anywhere.